dashboard/cron.py

```diff
@@ -385,14 +385,14 @@ def submission(self):
         # loop through multiple course ids
         for data_warehouse_course_id in Course.objects.get_supported_courses():
             submission_url = f"""with sub_fact as (select submission_id, assignment_id, course_id, user_id, global_canvas_id, published_score from submission_fact sf join user_dim u on sf.user_id = u.id where course_id = '{data_warehouse_course_id}'),
-                             enrollment as (select  distinct(user_id) from enrollment_dim where course_id = '{data_warehouse_course_id}' and workflow_state='active' and type = 'StudentEnrollment'),
-                             sub_with_enroll as (select sf.* from sub_fact sf join enrollment e on e.user_id = sf.user_id),
-                             submission_time as (select sd.id, sd.graded_at from submission_dim sd join sub_fact suf on sd.id=suf.submission_id),
-                             assign_fact as (select s.*,a.title from assignment_dim a join sub_with_enroll s on s.assignment_id=a.id where a.course_id='{data_warehouse_course_id}' and a.workflow_state='published' and muted = false),
-                             assign_sub_time as (select a.*, t.graded_at from assign_fact a join submission_time t on a.submission_id = t.id),
-                             all_assign_sub as (select submission_id AS id, assignment_id AS assignment_id, course_id, global_canvas_id AS user_id, round(published_score,1) AS score, graded_at AS graded_date from assign_sub_time order by assignment_id)
-                             select f.*, f1.avg_score from all_assign_sub f join (select assignment_id, round(avg(score),1) as avg_score from all_assign_sub group by assignment_id) as f1 on f.assignment_id = f1.assignment_id
-                             """
+                enrollment as (select  distinct(user_id) from enrollment_dim where course_id = '{data_warehouse_course_id}' and workflow_state='active' and type = 'StudentEnrollment'),
+                sub_with_enroll as (select sf.* from sub_fact sf join enrollment e on e.user_id = sf.user_id),
+                submission_time as (select sd.id, sd.graded_at, sd.posted_at at time zone 'utc' at time zone '{settings.TIME_ZONE}' as grade_posted_local_date from submission_dim sd join sub_fact suf on sd.id=suf.submission_id where posted_at is not null and posted_at < getdate()),
```

Comment:
>>> "sd.posted_at at time zone 'utc' at time zone '{settings.TIME_ZONE}' as grade_posted_local_date"

why there is a repeated "at time zone" phrase?

Reply:
Here is [Documentation](https://popsql.com/learn-sql/postgresql/how-to-convert-utc-to-local-time-zone-in-postgresql/) for it. I just took the approach from already code [here](https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/dashboard/cron.py#L363), Probably implemented by me in past

Reply:
It looks like you're changing this timezone from UTC to local time? Is there any reason you're doing this? It seems better just to keep everything in the database in UTC.

Reply:
That time has no use from the assignment view. The time when the grade was released seems to be not visible from Canvas UI for instructor/student. So I added that for tracking. I felt it is easier for @jennlove-um to test it time in local date rather than UTC.

Reply:
for `graded_at` we don't need it local time format. I used that as a clue if assignment is graded or not for color representation in UI

Reply:
Ok, I just suggested adding a comment to the model that the field was shifted to local time as part of the cron. If anyone used the model and tried to get the time it's going to be incorrect. 

We could store it as UTC (which would match the UDW) but then anyone using it would need to convert it in their queries.

Reply:
Done as per suggestion